v1/model.md

```diff
@@ -3,13 +3,15 @@
 This document lists all resource attributes that could be used when invoking the [Documaster API](endpoints.md). This document will help you get acquainted with the supported values for each resource in the following fields:
 - `data`
 - `expand`
+- `expandPageSize`
 - `attributes`
 - `sort`
  
 You can jump right to the section you are interested in:
 - [Model diagram](#model-diagram)
 - [Data](#data)
 - [Expand](#expand)
+- [Expand Page Size](#expandPageSize)
```

Comment:
The link has to be `#expand-page-size` to work as a link.

Reply:
Fix introduced with 41f8be6